app/modules/ads/ad-context.js

```diff
@@ -418,6 +418,10 @@ export const defaultAdContext = {
       enabled: false,
       propertyId: 'd-aIf3ibf0cYxCLB1HTWfBQOFEA',
     },
+    instantConfig: {
+      endpoint: 'https://services.wikia.com/icbm/api/config?app=mobile-wiki',
```

Comment:
what about fastboot?

Reply:
It isn't called on fastboot side. Instant config service is a part of ad-engine bundle (which is also not loaded on server side).